cypress/support/step_definition/course/deleteCourse/deleteCourese.spec.js

```diff
@@ -1,24 +1,24 @@
-import Commom_Logins from '../../../page/common/logins'
-import Delete_Course from '../../../page/course/deleteCoursePageObject'
-import Room_Overview from '../../../page/common/goToRoomOverview'
+import Commom_Logins from '../../../page/common/pageLogins'
+import Delete_Course from '../../../page/course/pageDeleteCoursePageObject'
+import Room_Overview from '../../../page/common/pageGoToRoomOverview'
+
+const commonLogin = new Commom_Logins()
+const roomOverview = new Room_Overview()
+const deleteCourse = new Delete_Course()
 
 
 Given('I am logged in as a teacher', () => {
```

Comment:
Same as for https://github.com/hpi-schul-cloud/e2e-system-tests/pull/9/files#diff-96bacab8462badc3e69ea60ed6a60d2fcdc8819b6623d3dcb21fc20337950dc4L6.